docs/core/versions/index.md

```diff
@@ -13,89 +13,220 @@ ms.assetid: f6f684b1-1d2c-4105-8376-7c1959e23803
+
+## Semantic Versioning
+
+.NET Core uses [Semantic Versioning (SemVer)](http://semver.org/), adopting the use of `MAJOR.MINOR.PATCH` versioning, using the various parts of the version number to describe the degree and kind of change.
+
+```
 MAJOR.MINOR.PATCH[-PRERELEASE-BUILDNUMBER]
+```
 
-### Decision Tree
+The optional `PRERELEASE` and `BUILDNUMBER` parts will never be part of supported releases, and will only exist on nightly builds, locally built from source targets, and unsupported preview releases.
 
-MAJOR when:
-  - drop support for a platform
-  - adopt a newer MAJOR version of an existing dependency 
-  - disable a compatibility quirk off by default
+### How do we increment version numbers?
 
-MINOR when:
-  - add public API surface area 
-  - add new behavior
-  - adopt a newer MINOR version of an existing dependency
-  - introduce a new dependency 
+We increment `MAJOR` when:
+  - we stop support for an old platform
+  - we adopt a newer `MAJOR` version of an existing dependency 
+  - we change the default setting of a compatibility quirk to "off"
+
+We increment `MINOR` when:
+  - we add public API surface area 
+  - we add new behavior
+  - we adopt a newer `MINOR` version of an existing dependency
+  - we introduce a new dependency 
   
-PATCH when:
-  - make bug fixes
-  - add support for a newer platform
-  - adopt a newer PATCH version of an existing dependency
-  - any other change (not otherwise captured)
+We increment `PATCH` when:
+  - we make bug fixes
+  - we add support for a newer platform
+  - we adopt a newer `PATCH` version of an existing dependency
+  - we make any other change that doesn't fit one of the above cases
+
+When there are multiple changes, we increment the highest element that would be affected by individual changes, and reset the following ones to zero. For instance, when `MAJOR` is incremented, `MINOR` and `PATCH` are reset to zero. When `MINOR` is incremented, `PATCH` is reset to zero while `MAJOR` is left untouched.
+
+### Preview versions
+
+Preview versions have a `-preview-[number]-([build]|"final)"` appended to the version, e.g. `2.0.0-preview-1-final`.
+
+### LTS vs. current
+
+We maintain two trains of releases for .NET Core, to enable users to pick the level of stability and new features they want, while being supported.
+
+- Long Term Support (LTS) means you'll get new features less frequently, but you'll have a more mature platform, that will be supported for a longer period.
+- Current means you'll get new features and APIs more frequently, but the flipside is that you'll have a shorter window of time to install updates, and those will happen more frequently. Current is also fully supported.
+
+A "current" version may get promoted to LTS.
```

Comment:
It might be worth mentioning that while "Current" is supported, older "Current" releases roll off support faster than LTS versions. I think that would be an important consideration for determining when to use "LTS" and when to use "Current".